src/routes/townhalls/index.ts

```diff
@@ -211,7 +211,8 @@ router.post<
     makeEndpoint(async (req, res) => {
         // TODO: addresses fixme, but check if the user is an organizer based off the api token
         const { email, firstName, lastName } = req.body;
-        await register(email, firstName, lastName);
+        const { townhallId } = req.params;
```

Comment:
The typechecking is my fault.  Just comment out both of the breakout endpoints to fix the linting/typechecking.